goal_src/jak2/engine/ui/text-id-h.gc

```diff
@@ -719,6 +719,11 @@
   (progress-hires-sky #x1309)
   (progress-fast-airlock #x130a)
   (progress-fast-elevator #x130b)
+  (progress-toggle-input #x130c)
+  (pad-triangle #x2000)
+  (pad-circle #x2001)
+  (pad-x #x2002)
+  (pad-square #x2003)
```

Comment:
Add these entries to the enum in `all-types.gc` so the type consistency test passes.